pkg/daemon/ceph/client/mirror.go

```diff
@@ -50,8 +51,9 @@ type Images struct {
 }
 
 var (
-	rbdMirrorPeerCaps      = []string{"mon", "profile rbd-mirror-peer", "osd", "profile rbd"}
-	rbdMirrorPeerKeyringID = "rbd-mirror-peer"
+	rbdMirrorPeerCaps                                         = []string{"mon", "profile rbd-mirror-peer", "osd", "profile rbd"}
+	rbdMirrorPeerKeyringID                                    = "rbd-mirror-peer"
+	cephBlockPoolRadosNamespaceMirroringMinimumFeatureVersion = cephver.CephVersion{Major: 20, Minor: 0, Extra: 0}
```

Comment:
nit: How about shortening the name? We don't need the full crd name in a var name.
```suggestion
	radosNamespaceMirroringMinimumVersion = cephver.CephVersion{Major: 20, Minor: 0, Extra: 0}
```